frontend/env_config/dev/config.json

```diff
@@ -1,4 +1,4 @@
 {
-  "idps_to_show": "idir,moh_idp",
```

Comment:
Unrelated change. We never reference `idps_to_show` so I removed it from all the config files.

Reply:
We were using it locally for the testcafe user to be able to login I think